BUILD.md

```diff
@@ -215,6 +215,14 @@ make -j4
 For build systems that support ccache, enable it with the CMake
 `-DUSE_CCACHE=On` option.
 
+## Install the project
+
+Files can be installed to "/usr/local/" with `sudo make install`
+
+For Linux, the defualt install directory `/usr/local` can be changed by setting CMake's CMAKE_INSTALL_PREFIX
```

Comment:
There is a typo here with the spelling of `default`

Reply:
Just pushed a fix